### ISSUE TITLE
In elementor form, set the honeypot as a requirement by default ...

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#17629]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

hello

On my websites, sending spam through the elementor form was very common. Maybe 15 or 20 spam a week. At least.
It was really painful. Because when I activated HoneyPot in elementor form, the spam sent continued. In other words, honeypot was of no use.
Finally, using a trick, I made the honeypot option requirement. The problem was solved! Spam is no longer sent to the elementor form.
In other words, the problem is that honeypot is not currently a requirement. Therefore, robots do not intelligently fill it and are known as normal users (humans).
Using the following method, I made honeypot a requirement. (It does not seem logical, but that's the problem)

https://user-images.githubusercontent.com/77163248/151695049-dcac604d-3093-4ecd-bf9d-03584e7d5b98.mp4

### Describe the solution you'd like

Nothing, just set the Honey Pot as a requirement by default. So that the robots have to fill it.

### Additional context

I was not sure if I should register this as a new issues. I was not sure if this included a bug, or a small new request.

thank you
